lib/broccoli/amd-shim.js

```diff
@@ -6,11 +6,14 @@ module.exports = function shimAmd(tree, nameMapping) {
     let name = nameMapping[relativePath];
     if (name) {
       return [
-        '(function(define){\n',
+        '(function(define, module){\n',
```

Comment:
Sorry to be overly nit-picky here, but it would be _wonderful_ if we could clean this up with a bit of ES6 syntax.

* Use template strings with interpolation for `content`
* spread out for readability

Might look like:

```js
return `
(function(define, module){

${content}

  if (define.defined === false && module.exports) { 
    define([], function () { 
      return module.exports; 
    });
  }
})((function(){ 
  function newDefine(){ 
    var args = Array.prototype.slice.call(arguments);
    while (typeof args[0] === "string") { args.shift(); }
    args.unshift("${name}");
    define.defined = true; 
    return define.apply(null, args);
  }; 

  newDefine.amd = true; 
  newDefine.defined = false; 

  return newDefine;
})(), {});
`
```